### PR TITLE
Yf/apollo improvements

### PR DESCRIPTION
**Pull request by WildFireFlum**
Added support for seeded test runs, seeds are printed to Apollo's log.
Added support for printing Apollo logs to stdout using the APOLLO_LOG_STDOUT environment variable.
Removed some unused global variables.

### CODE REVIEW

File: tests/config/test_comm_config.cpp [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 #include "communication/CommFactory.hpp"
 #include "KeyfileIOUtils.hpp"
-#include "config_file_parser.hpp"
+#include "../../util/include/config_file_parser.hpp"
```
Comment (yontyon) [inline tests/config/test_comm_config.cpp:25]: don't use relative paths in include, if needed, add the h file to the cmake file

**Review by yontyon — APPROVED** [review]
LGTM!
Thanks

**Review by KTeofanidis — APPROVED** [review]
LGTM

File: tests/config/test_comm_config.cpp (hunk 1 above)
Comment (WildFireFlum) [reply tests/config/test_comm_config.cpp]: Done

**Pull request merged**